.gitignore

```diff
@@ -82,11 +83,11 @@ celerybeat-schedule
 *.sage.py
 
 # Environments
-.env
+examples/.env
```

Comment:
Probably no harm in leaving the .env too, or if there's a way to do both. Might stop Muppets like me from accidentally committing creds if I use the wrong folder